ministryofjustice/parliamentary-questions
# CDPT-873 add brakeman checks to Circle CI pipeline

## Description

Setup the Brakeman gem to run security checks with each Circle CI build.
 
## Self-review checklist


### Screenshots

N/A

### Related JIRA tickets

https://dsdmoj.atlassian.net/jira/software/c/projects/CDPT/boards/1152?selectedIssue=CDPT-873

### Deployment

N/A

### Manual testing instructions

As the checks are run during the build process a dev should be able to show the build output.  please also check the 'happy path' for anything unusual.
